src/urllib3/connectionpool.py

```diff
@@ -544,17 +544,14 @@ def _make_request(
         response._connection = response_conn  # type: ignore[attr-defined]
         response._pool = self  # type: ignore[attr-defined]
 
-        # emscripten connection doesn't have _http_vsn_str
-        http_version = getattr(conn, "_http_vsn_str", "HTTP/?")
         log.debug(
-            '%s://%s:%s "%s %s %s" %s %s',
+            '%s://%s:%s "%s %s HTTP/%s" %s %s',
             self.scheme,
             self.host,
             self.port,
             method,
             url,
-            # HTTP version
-            http_version,
+            response.version,
```

Comment:
Thoughts on using `*divmod(response.version, 10)`?

Reply:
It would give HTTP/2.0 which is less wrong but still weird? I think I would prefer storing the version string in the response? It can be done without any computation and only costs a few bytes?

Reply:
Yeah, I do like HTTP/2 in the logs better too.

Reply:
Thinking about this, we should probably keep emitting `HTTP/1.1` and `HTTP/2` since some folks might be using regular expressions on these values. Can we have an `if` block that determines the value based on the `resp.version` value?

Reply:
There's a way to avoid `if`s, what do you think of https://github.com/urllib3/urllib3/pull/3316/commits/a1f18b0166e84b83e72565d0e4f053e61fa97653?

Reply:
> Thinking about this, we should probably keep emitting `HTTP/1.1` and `HTTP/2` since some folks might be using regular expressions on these values. Can we have an `if` block that determines the value based on the `resp.version` value?

I am one who had tests that looked for the entire debug log line with the HTTP/1.1 and now they are breaking on version 2.2.2. Leaving the dot out for version 1.1 is invalid so it would be better for the long haul to bring that back.

Reply:
What are you seeing exactly? It was supposed to still be HTTP/1.1.

Reply:
I'm not specifying everything in my requirements file, but when 2.2.2 became available and was installed I see.
> https://localhost:8082 "PUT /activeUser?force=true HTTP/11" 200 4

When I explicitly revert to 2.2.1 in the requirements file and rebuild the environment I see the previous behavior.
> https://localhost:8082 "PUT /activeUser?force=true HTTP/1.1" 200 4

I can open a issue.